### PR TITLE
Add comment about local development

### PR DESCRIPTION
**Pull request by weebl2000**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -408,6 +408,8 @@ Development
 
 run *ghcid* with: ``ghcid -c "cabal repl exe:cardano-node --reorder-goals"``
 
+Note: It might be required to set ``ghc-options: -Wwarn`` in `cabal.project` when developing on modules. Otherwise GHC complains about packages specified that are not required.
```
Comment (dcoutts) [inline README.rst:411]: It'd be better to recommend setting the existing `development` flags in the `cabal.project` file, e.g.
```
package shelley-spec-ledger
  flags: +development
```
This is what the top level `cardano-haskell` `cabal.project` file does for example.

https://github.com/input-output-hk/cardano-haskell/

Comment (weebl2000) [reply README.rst:411]: I will test if this works and change the comment.

Comment (weebl2000) [reply README.rst:411]: image

I don't think it's sufficient.

Comment (weebl2000) [reply README.rst]: @dcoutts I've amended the readme to be more specific and added the ``development`` flag as you suggested.

**Comment by weebl2000** [comment]
@Jimbo4350 Sorry for tag, but not sure if Duncan is still involved in the project. Small comment about developing locally that might help people. Suggestions have been processed.

**Comment by weebl2000** [comment]
@coot Sorry for shameless tagging. But this was basically approved. Can we merge it?

**Review by olgahryniuk — APPROVED** [review]

**Comment by coot** [comment]
I prefer the other way around, remove `-Werror` and only use it in CI and for those who want to have it in their dev cycle, set `-Werror` in `cabal.project.local` file.

**Review by newhoggy — APPROVED** [review]

**Pull request merged**